RecoLocalTracker/SiPixelRecHits/interface/pixelCPEforGPU.h

```diff
@@ -12,6 +12,17 @@
 #include "HeterogeneousCore/CUDAUtilities/interface/cudaCompat.h"
 #include "CUDADataFormats/TrackingRecHit/interface/SiPixelHitStatus.h"
 
+namespace CPEFastParametrisation {
+  // From https://cmssdt.cern.ch/dxr/CMSSW/source/CondFormats/SiPixelTransient/src/SiPixelGenError.cc#485-486
+  // qbin: int (0-4) describing the charge of the cluster
+  // [0: 1.5<Q/Qavg, 1: 1<Q/Qavg<1.5, 2: 0.85<Q/Qavg<1, 3: 0.95Qmin<Q<0.85Qavg, 4: Q<0.95Qmin]
+  constexpr int GenErrorQBins = 5;
+  // arbitrary number of bins for sampling errors
+  constexpr int NumErrorBins = 16;
```

Comment:
```suggestion
  constexpr int kGenErrorQBins = 5;
  // arbitrary number of bins for sampling errors
  constexpr int kNumErrorBins = 16;
```
- start local variables and data members with lower case letter. Capitalization is reserved for class names.
- Also, a somewhat recent rule for constants http://cms-sw.github.io/cms_coding_rules.html#2--naming-rules-1 2.17 (this one is up for debate for this code, considering the style for other constants in the related code are likely to be the exception to this rule).